shared/common-adapters/animation-data.json

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
note, the old pr didn't work with storyshots due to this naming collision. it saw animation.json and thought it shoudl load that. so in general we want to just avoid that so renaming this animation-data.json
cc: @adamjspooner